test/parallel/test-util-types.js

```diff
@@ -1,4 +1,5 @@
-// Flags: --experimental-vm-modules --expose-internals --allow-natives-syntax
+// Flags: --experimental-vm-modules --expose-internals --allow-natives-syntax --js-float16array
+// TODO(LiviaMedeiros): once `Float16Array` is unflagged in v8, remove `--js-float16array` above
```

Comment:
I also think we shouldn't land this until this becomes stable.

Reply:
I think doing it the other way around is actually good so that the new method is available as soon as it becomes stable. It will likely not be used before that anyway.

Reply:
It is already stable, it is stage 4 – and if you meant the V8 implementation, that's really not necessary IMO, precisely because we can trust it will follow the spec.